### PR TITLE
As a guest, I can see a visual representation of user and media review scores.

### PR DESCRIPTION
**Pull request by reshleman**
**Visually Display Review Scores:**
- Display review scores instead of placeholder images
- Extract helpers for review score `div`s
- Extract helpers to for captioned (averaged) review scores
- Apply CSS class to review score based on value
- Update CSS to display colored square with review score
- Add default (gray) color to CSS for review scores

**Refactorings:**
- Extract CSS for review scores to separate file
- Rename `review_statistics_helper` to `reviews_helper`
- Rename score formatting helpers for clarity (and update views)
- Change CSS selectors for `review-image` to `review-score`

https://trello.com/c/AJntujhL

### CODE REVIEW

File: app/helpers/reviews_helper.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    case score.to_f
+    when 0.0...4.0 then "review-score-negative"
+    when 4.0...7.0 then "review-score-neutral"
+    when 7.0..10.0 then "review-score-positive"
+    else "review-score"
+    end
+  end
+
+  def media_review_css_class(score)
+    case score.to_f
```
Comment (halogenandtoast) [inline app/helpers/reviews_helper.rb:59]: This is a case for VALUE OBJECTS!

File: app/helpers/reviews_helper.rb [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  end
+
+  def format_user_review_score(score)
+    format_review_score(score || 0)
+  end
+
+  private
+
+  def user_review_css_class(score)
+    case score.to_f
```
Comment (halogenandtoast) [inline app/helpers/reviews_helper.rb:50]: This is a case for VALUE OBJECTS!

File: app/assets/stylesheets/_review_scores.scss [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+.review-score-neutral {
+  @extend .review-score;
+  background-color: $review-score-neutral-color;
+}
+
+.review-score-negative {
+  @extend .review-score;
+  background-color: $review-score-negative-color;
+}
+
```
Comment (halogenandtoast) [inline app/assets/stylesheets/_review_scores.scss]: Hello newline.

File: app/helpers/reviews_helper.rb (hunk 1 above)
Comment (reshleman) [inline app/helpers/reviews_helper.rb:59]: @halogenandtoast Thanks, I'll give that refactoring a try as a separate PR.

**Pull request closed without merging**